sites/cheerpx/src/pages/index.astro

```diff
@@ -1,60 +1,243 @@
-			frameborder="0"
-			allowfullscreen="allowfullscreen"></iframe>
-	</section>
-</ProductHome>
 
-<style>
-	@keyframes flyFadeScaleIn {
-		0% {
-			opacity: 0;
-			transform: translateY(70px) scale(0.6);
-		}
-		100% {
-			opacity: 1;
-			transform: translateY(0) scale(1);
-		}
-	}
+	<!-- TODO: video -->
+	<!-- TODO: client company logos -->
+	<!-- TODO: how it works (user story) -->
+	<!-- TODO: testimonials -->
+	<!-- TODO: FAQ -->
 
-	.fly-in-b {
-		animation: flyFadeScaleIn 300ms cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
-	}
-</style>
+	<section
+		id="gamesrunner"
+		class="bg-gradient-to-b from-black to-black via-stone-900 py-32 px-8"
+	>
+		<div class="items-center px-4 md:px-8 lg:px-16 xl:px-32 2xl:px-32"></div>
+		<div
+			class="flex grid grid-cols-1 px-4 sm:grid-cols-1 md:grid-cols-1 xl:grid-cols-3 px-4 md:px-8 lg:px-16 xl:px-32 2xl:px-56"
+		>
+			<div class="col-span-2">
+				<Image src={GamePadAsset} alt="Game pad" class="w-32 sm:w-48" />
+				<div class=""></div>
+				<h1 class="font-semibold text-3xl sm:text-4xl lg:text-5xl py-2">
+					CheerpX Games Runner
+				</h1>
+				<p class="text-xl sm:text-xl">
+					CheerpX Games Runner is an open-source Chrome extension to run your
+					games directly in the browser.
+				</p>
+				<ul class="list-inside list-disc font-normal text-wrap text-xl py-4">
+					<li>
+						Play historical games from your collection directly in your browser.
+					</li>
+					<li>
+						Works on any platform, even if not officially supported by the game.
+					</li>
+					<li>No local installation required.</li>
+				</ul>
+				<div class="flex gap-4 justify-left py-2">
+					<LinkButton
+						type="primary-cheerpx"
+						href="https://chromewebstore.google.com/detail/cheerpx-games-runner-beta/kpjhccfibjgklihcmaddjecaenppaahg"
+						label="Install now"
+						iconRight="teenyicons:chrome-solid"
+					/>
+					<LinkButton
+						type="secondary-cheerpx"
+						href="https://github.com/leaningtech/cheerpx-games-runner"
+						iconRight="bi:github"
+						label="Contribute"
+					/>
+				</div>
+			</div>
+			<div class="col-span-1 flex items-center sm:items-end px-4">
+				<Image src={Screenshot} alt="screenshots" class="" />
+			</div>
+		</div>
+	</section>
+	<section id="youtube" class="bg-black py-32 px-8">
+		<div
+			class="grid grid-cols-1 items-center sm:grid-cols-1 xl:grid-cols-2 px-4 sm:px-8 md:px-16 lg:px-28 xl:px-32 2xl:px-48"
+		>
+			<div class="w-full col-span-1">
+				<iframe
+					title="CheerpX: a WebAssembly-based x86 virtual machine in the browser, Yuri Iozzelli"
+					src="https://youtube.com/embed/VqrbVycTXmw?start=151"
+					class="w-full max-w-[1080px] aspect-video rounded-xl shadow-lg border-2 border-stone-900"
+					frameborder="0"
+					allowfullscreen="allowfullscreen"
+				>
+				</iframe>
+			</div>
+			<div class="col-span-1 pl-8">
+				<h1
+					class="text-white text-5xl xl:text-5xl 2xl:text-6xl font-semibold py-4 text-left text-wrap md:text-balance"
+				>
```

Comment:
We might need to get the text here to be centralised rather than left-aligned on the phone screen to show it fully. Or alternatively we can reduce the size of the h1 font here

Reply:
Good idea, thanks. This whole section will be different, I'll be removing the youtube video, probably I will leave the title in a slightly different spread. Thanks for the review!